packages/react-components/react-tags-preview/stories/TagGroup/TagGroupSizes.stories.tsx

```diff
@@ -26,7 +26,7 @@ export const Sizes = () => {
       {sizes.map(size => (
         <div key={size}>
           {`${size}: `}
-          <TagGroup size={size}>
+          <TagGroup size={size} aria-label="Sizes">
```

Comment:
is this really an accessible label? what should I interpret from this control with `Sizes`? 😅

Reply:
:D Oops, let me update it